tensorflow_datasets/audio/freesound_audio_tagging_2019_test.py

```diff
@@ -0,0 +1,74 @@
+# coding=utf-8
+# Copyright 2019 The TensorFlow Datasets Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Tests for Freesound Audio Tagging 2019 dataset module."""
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+from multiprocessing import cpu_count
+
+import tensorflow as tf
+
+from tensorflow_datasets import testing
+from tensorflow_datasets.audio import freesound_audio_tagging_2019
+
+
+def test_spectrogram():
+    def preprocess(row):
+        audio_path = row['audio']
+        blob = tf.io.read_file(audio_path)
+
+        waveform = tf.audio.decode_wav(blob, 1, 3.0 * 44100)[0]
+
+        x = tf.transpose(waveform)
+        x = tf.signal.stft(x, 1024, 512)
+        x = tf.transpose(x, (1, 2, 0))
+        spectrogram = tf.abs(x)
+
+        tags = row['tags']
+        tags = tf.stack(list(tags.values()))
+
+        return spectrogram, tags
+
+    dataset = tfds.load(
+        name='freesound_audio_tagging2019',
```

Comment:
You should use fake examples to testing. Check this [link](https://github.com/tensorflow/datasets/blob/master/docs/add_dataset.md#testing-mydataset).